portfolio/src/main/webapp/skills.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE HTML>
+<html lang = "en">
+   <head>
+      <!-- basic.html -->
+      <title>Skills Page</title>
+      <meta charset = "UTF-8" />
+      <meta name="viewport" content = "width=device-width, initial-scale=1.0">
+      <meta http-equiv = "X-UA-Compatible" content="ie=edge">
+      <link rel="stylesheet" href = "style/format.css">
+   </head>
+   <body class="menu-page page">
+      <div class = "skills-background">
```

Comment:
there are still some formatting inconsistencies here and also in other HTML files :(

Reply:
I believe the formatting is fixed now

Reply:
I think the formatting is still off in a few files :(

Reply:
Just formatted with the online formatter